src/pathfora.js

```diff
@@ -3593,6 +3607,17 @@
           context.pathfora.closeWidget(widget.id, true);
         }, widget.displayConditions.hideAfter * 1000);
       }
+
+      core.widgetResizeListener(widget, node);
+
+      var resize = window.onresize;
+      window.onresize = function (event) {
+        if (typeof resize === 'function') {
```

Comment:
I don't think so. The idea was to preserve the `window.onresize` function if it had already been set by the user elsewhere in javascript. So if it hasn't `resize` will be `null`. And we skip calling `resize(event)`. I tested it and I think that's how it should work. It does look kind of strange though...